### PR TITLE
[AAE-6310] Fix for attaching files

### PR DESCRIPTION
**Pull request by BSekula**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-6310

After navigate to process page with query params for process i.e. `process="my-proess-name"` form and attached files were not shown   

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html [hunk 1]
```diff
@@ -19,9 +19,17 @@
...
+                            #auto="matAutocomplete"
+                            id="processDefinitionOptions"
+                            [displayWith]="displayProcessNameOnDropdown"
+                            (optionSelected)="setProcessDefinitionOnForm($event.option.value)"
+                        >
+                            <mat-option
+                                *ngFor="let processDef of filteredProcesses"
+                                [value]="getProcessDefinitionValue(processDef)"
+                                (click)="processDefinitionSelectionChanged(processDef)"
+                            >
```
Comment (BSekula) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html:32]: Indention only

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts [hunk 2]
```diff
@@ -247,16 +243,24 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
...
+                    this.selectDefaultProcessDefinition();
+                } else if (this.processDefinitionName) {
+                    this.processDefinition.setValue(this.processDefinitionName);
+
+                    const processDefinition = this.processDefinitionList.find(process => process.name === this.processDefinitionName);
+                    if (processDefinition) {
+                        this.filteredProcesses = this.getProcessDefinitionListByNameOrKey(processDefinition.name);
+                        this.setProcessDefinitionOnForm(processDefinition.name);
+                        this.processDefinitionSelectionChanged(processDefinition);
                     }
```
Comment (BSekula) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts:257]: Only added part here is:

```
const processDefinition = this.processDefinitionList.find(process => process.name === this.processDefinitionName);
  if (processDefinition) {
    this.filteredProcesses = this.getProcessDefinitionListByNameOrKey(processDefinition.name);
    this.setProcessDefinitionOnForm(processDefinition.name);
    this.processDefinitionSelectionChanged(processDefinition);
}
```
rest are indentations

**Review by Qetikv — APPROVED** [review]

**Comment by eromano** [comment]
@BSekula can we avoid the indentation change? thanks

**Review by eromano — CHANGES_REQUESTED** [review]
commented added

**Comment by BSekula** [comment]
reverted indentation changes

**Review by eromano — APPROVED** [review]

**Pull request merged**